src/main/java/org/mockito/internal/stubbing/InvocationContainerImpl.java

```diff
@@ -121,6 +121,10 @@ public String toString() {
         return registeredInvocations.getAll();
     }
 
+    public void removeInvocations() {
+        registeredInvocations.clear();
```

Comment:
I like the notion of _clearing_, I propose the pull this word up to the `Mockito` class facade. instead of using 3 different words (reset, clear, remove).